build.sbt

```
@@ -1,8 +1,8 @@
 organization := "org.http4s"
 name := "jawn-fs2"
 
-scalaVersion := "2.13.1"
-crossScalaVersions := Seq("2.12.10", scalaVersion.value)
+ThisBuild / scalaVersion := "2.13.1"
+ThisBuild / crossScalaVersions := Seq("2.12.10", scalaVersion.value)
```

Don't do this! Never do this. It breaks the moment anyone does `++`.

You're much better off inverting this dependency:

```suggestion
ThisBuild / crossScalaVersions := Seq("2.12.10", "2.13.1")
ThisBuild / scalaVersion := crossScalaVersions.value.last
```

---

:face_with_head_bandage: